### DISCUSSION TITLE
Slash on slug field

### DISCUSSION THREAD

**Discussion by ivoreis82** [directus/directus#3710] [Feature Requests]
When the slug option is selected it should be possible to add a slash.

**Comment by philleepflorence** [comment]
Do you mean adding a slash to replace the hyphen or adding a prefix/suffix.
E.G. 
`/prefix/this-is-the-slug/suffix/`

**Reply by ivoreis82** [reply]
adding a prefix/suffix

**Comment by pwFoo** [comment]
Would be nice to create computed slug fields for example to create page path 

pages collection:
- id
- title
- parent (reference to parent)
- name (slug of title column or text input page name like "Home" -> slug version "home"?)
- path (computed slug of parent.path and column name: "<parent_path>/<page_name>")

Would that possible from inside of the adminstration ui with something like "collection.column" variables?
Slug "code" could be `{parent.path}/{name}` (if no parent computed result should be `/{name}`)

**Reply by pwFoo** [reply]
My question with computed field values is related to https://github.com/directus/directus/discussions/4231 discussion.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
